config/main.py

```diff
@@ -353,11 +353,9 @@ def mirror_session():
 @click.argument('session_name', metavar='<session_name>', required=True)
 @click.argument('src_ip', metavar='<src_ip>', required=True)
 @click.argument('dst_ip', metavar='<dst_ip>', required=True)
-@click.argument('gre_type', metavar='<gre_type>', required=True)
```

Comment:
can you change to not required, if not specified by the user, you do not need that field.